airbytehq/airbyte
# Change to trigger CI

## What
Follow-up on https://github.com/airbytehq/airbyte/pull/40713, addresses https://github.com/airbytehq/airbyte-internal-issues/issues/8447

On CDK changes that can modify the binaries, run source-shopify tests.

## How

By adding a new GitHub workflow that will get triggered if there are any changes on the following files
 - 'airbyte-cdk/python/airbyte_cdk/**'
 - 'airbyte-cdk/python/bin/**'
- 'airbyte-cdk/python/poetry.lock'
- 'airbyte-cdk/python/pyproject.toml'

Manual tests performed:
* The CI check is marked as green in 4 seconds when the pull request does not include CDK changes
* Change in the CDK trigger testing for source-shopify
* (currently not behaving as expected (see print screen)) Pushing a new change stops the current tests
![image](https://github.com/airbytehq/airbyte/assets/3360483/fd8fb527-4933-4a4b-ba2f-7f621ca78c1c)

Things I haven't tested and assume works:
* use-local-cdk flag

## User Impact
The devs will now know if their CDK changes are working for source shopify. Eventually, we will add more connectors to that

## Can this PR be safely reverted and rolled back?
